apps/snitch_core/lib/core/data/schema/order.ex

```diff
@@ -72,18 +72,18 @@ defmodule Snitch.Data.Schema.Order do
   def update_changeset(%__MODULE__{} = order, params) do
     order
     |> cast(params, @update_fields)
-    |> common_changeset()
     |> cast_assoc(:line_items, with: &LineItem.create_changeset/2)
     |> ensure_unique_line_items()
     |> compute_totals()
   end
 
-  @spec common_changeset(Ecto.Changeset.t()) :: Ecto.Changeset.t()
-  defp common_changeset(order_changeset) do
-    order_changeset
-    |> unique_constraint(:number)
-    |> foreign_key_constraint(:billing_address_id)
-    |> foreign_key_constraint(:shipping_address_id)
+  @partial_update_fields ~w(state)a
+  @spec partial_update_changeset(t, map) :: Ecto.Changeset.t()
+  def partial_update_changeset(%__MODULE__{} = order, params) do
```

Comment:
```diff
+  alias Snitch.Data.Schema.Order
-  def partial_update_changeset(%__MODULE__{} = order, params) do 
+  def partial_update_changeset(%Order{} = order, params) do 
```
Instead of using `__MODULE__` which confuses while looking at the line what this struct is and again we have to look at the name of the module to see what the name of the `struct`. 

This is just a Developer Experience.

Reply:
I was merely following our [style guide](https://github.com/lexmag/elixir-style-guide#modules). `__MODULE__` is quite unambiguous IMO, it's the file that you are reading :smile: